core/templates/dev/head/components/question-directives/questions-list/questions-list.constants.ts

```diff
@@ -16,7 +16,10 @@
  * @fileoverview Constants for the question player directive.
  */
 
-angular.module('oppia').constant('DEFAULT_SKILL_DIFFICULTY', '0.3');
-angular.module('oppia').constant(
-  'MODE_SELECT_DIFFICULTY', 'MODE_SELECT_DIFFICULTY');
-angular.module('oppia').constant('MODE_SELECT_SKILL', 'MODE_SELECT_SKILL');
+export class QuestionsListConstants {
+  public static DEFAULT_SKILL_DIFFICULTY = 0.3
```

Comment:
Ditto, inconsistencies in trailing semicolons. Please look out for these elsewhere.

@kevinlee12 we should perhaps turn on a lint check in eslint for this if there's one.